core/backend/src/ClassRegistry.ts

```diff
@@ -17,13 +17,72 @@ import { _nativeDb } from "./internal/Symbols";
 
 const isGeneratedClassTag = Symbol("isGeneratedClassTag");
 
+/** Maintains the mapping between the name of a BIS [ECClass]($ecschema-metadata) (in "schema:class" format) and the JavaScript [[Entity]] class that implements it.
+ * @public
```

Comment:
this API is going straight to public?

Reply:
You spotted the controversial thing in this PR :-)
Arun mentioned the same. We discussed internally about this. We want to mark all of ecschema-metadata as "stable for one release" for the 5.0 timeframe, but that mechanism is not stable yet, so it's still marked as beta.
So really, beta and public are our only two options here. As soon as the mechanism is available, we will mark this as "stable for one release".
My rationale for choosing public for now was that we are deprecating existing public methods, so our replacements being marked as beta just doesn't seem right.